content/docs/hoon/reference/rune/bar.md

```diff
@@ -4,118 +4,147 @@ weight = 7
-with changes, which is useful for recursion among other things.
+Five core runes (`|=`, `|.`, `|-`, `|*`, and `|$`) produce a core with a single
+arm, named `$`. As with all arms, we can recompute `$` with changes, which is
+useful for recursion among other things.
 
-## Runes
+## `|$` "barbuc"
 
-### |$ "barbuc"
+Declares a mold builder wet gate with one or more molds as its sample.
 
-Declares a mold builder wet gate with one or more molds as its
-sample.
+#### Syntax
 
-##### Syntax
+Two arguments, fixed.
 
-Regular:
-```hoon
-|$  a=(lest term)  b=spec
-```
+<table>
+<tr><th>Form</th><th>Syntax</th></tr>
+<tr>
+<td>Tall</td>
+<td>
+<pre>
+|$  sample
+body
+</pre>
+</td>
+</tr>
+<tr>
+<td>Wide</td>
+<td>
+<pre>
+|$(sample body)
+</pre>
+</td>
+</tr>
+<tr>
+<td>Irregular</td>
+<td>None.</td>
+</tr>
+</table>
+
+#### AST
 
-AST:
 ```hoon
-[%brbs sample=(lest term) body=spec]
+[%brbc sample=(lest term) body=spec]
 ```
 
-##### Semantics
-
-`|$` is used to declare a wet gate mold builder that is polymorphic in its input molds.  `a` is a `lest` of `term` used as
-identifiers for the input molds. `b` is a structure built from elements of `a`.
-The output of `|$` is a mold builder obtained by
-substituting the input molds parameterized by `a` into `b`.
-
-##### Expands to
+#### Expands to
 
 ```hoon
 |$  [a b]
 body
 ```
+
 becomes
+
 ```hoon
 |*  [a=$~(* $-(* *)) b=$~(* $-(* *))]
 ^:
 body
 ```
 
-##### Discussion
+#### Semantics
+
+`|$` is used to declare a wet gate mold builder that is polymorphic in its input
+molds. `a` is a `lest` of `term` used as identifiers for the input molds. `b` is
+a structure built from elements of `a`. The output of `|$` is a mold builder
+obtained by substituting the input molds parameterized by `a` into `b`.
+
+#### Discussion
 
 A mold builder is a wet gate from one or more molds to a mold. A mold is a
 function from nouns to nouns with types that may be partial, is always
 idempotent, and usually the identity function on the noun itself.
 
-`|$` is a restricted form of `|*`. The use of `|$` over `|*` is one of style, as either could
-be used to make wet gates that are mold builders. The buc in `|$` is a hint that
-`|$` is closely related to buc runes, and thus `|$` should be used to make mold builders, while `|*` should be used for any other sort of
-wet gate. Thus, the second argument of `|$` is frequently a buc
-rune. For further discussion of wet gates, see the entry for [`|*`](#bartar).
+`|$` is a restricted form of `|*`. The use of `|$` over `|*` is one of style, as
+either could be used to make wet gates that are mold builders. The buc in `|$`
+is a hint that `|$` is closely related to buc runes, and thus `|$` should be
+used to make mold builders, while `|*` should be used for any other sort of wet
+gate. Thus, the second argument of `|$` is frequently a buc rune. For further
+discussion of wet gates, see the entry for [`|*`](#-bartar).
```

Comment:
Another difference from `|*` is that the body of `|$` is parsed in pattern mode, to a `spec` (this is in your expansion, but might be worth explicitly noting here.